# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome14/oaf10/bison_syntax.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ru oaf-0.6.10_/oafd/ac-query-parse.y oaf-0.6.10/oafd/ac-query-parse.y
--- oaf-0.6.10_/oafd/ac-query-parse.y	2001-10-24 23:15:13.000000000 +0200
+++ oaf-0.6.10/oafd/ac-query-parse.y	2002-11-16 14:35:53.000000000 +0100
@@ -155,7 +155,7 @@
 stringlist: P_CONST_STRING { $$ = g_slist_prepend (NULL, $1); }
 	| stringlist COMMA P_CONST_STRING { $$ = g_slist_append ($1, $3); };
 
-expr_variable: P_DOLLAR P_CONST_ID { $$ = qexp_variable_new ($2); }
+expr_variable: P_DOLLAR P_CONST_ID { $$ = qexp_variable_new ($2); };
 
 expr_function: P_CONST_ID LPAREN exprlist RPAREN { $$ = qexp_function_new ($1, $3); }
 	| P_CONST_ID LPAREN RPAREN { $$ = qexp_function_new ($1, NULL); }
Only in oaf-0.6.10/oafd: ac-query-parse.y~
